{"uuid":"1810f251-6e63-4737-8f36-d46db07999a5","historyId":"5100df7a14ad70b8799d44a2f2546bb2","fullName":"testcases.LoginPageTest.testLoginWithoutInvalidCredentials","labels":[{"name":"package","value":"testcases.LoginPageTest"},{"name":"testClass","value":"testcases.LoginPageTest"},{"name":"testMethod","value":"testLoginWithoutInvalidCredentials"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"E:\\PerkingAutomation\\src\\test\\java\\testcases"},{"name":"subSuite","value":"testcases.LoginPageTest"},{"name":"host","value":"Hride"},{"name":"thread","value":"14876@Hride.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testLoginWithoutInvalidCredentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Invalid Login Credentials! ] but found [Invalid Login Credentials!]","trace":"java.lang.AssertionError: expected [Invalid Login Credentials! ] but found [Invalid Login Credentials!]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat testcases.LoginPageTest.testLoginWithoutInvalidCredentials(LoginPageTest.java:33)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Testing login functionality with invalid credentials","steps":[],"attachments":[{"name":"After trying to logging in with wrong credentials","source":"ddd6786b-5579-4dd9-a2c9-b86354ee458e-attachment"}],"parameters":[{"name":"arg2","value":"Invalid Login Credentials! "},{"name":"arg1","value":"12345"},{"name":"arg0","value":"Tausif"}],"start":1716180369646,"stop":1716180372572}